tutorials/cp-workflow-getting-started/cp-workflow-getting-started.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Prepare to create workflows in SAP Cloud Platform
+description: Enable and configure the services involved and assign the user roles you'll need for workflows.
+primary_tag: products>sap-cloud-platform
+tags: [  tutorial>beginner, topic>cloud, products>sap-cloud-platform ]
+---
+
+## Prerequisites  
+ - **Proficiency:** Beginner
+ - **Tutorials:** [Sign up for a free trial account on SAP Cloud Platform](https://www.sap.com/developer/tutorials/hcp-create-trial-account.html)
+
+## Details
+### You will learn  
+- To enable the workflow service in your SAP Cloud Platform account.
+- To assign the roles you need to develop and use workflow applications.
+
+### Time to Complete
+**5 Min**
+
+---
+
+[ACCORDION-BEGIN [Step 1: ](Open the SAP Cloud Platform Cockpit)]
+
+1. In your Web browser, open the [SAP Cloud Platform](https://account.hanatrial.ondemand.com/cockpit) cockpit. If you do not have a trial account, see Prerequisites.
```

Comment:
Are there prerequisites that were deleted?  I don't see a section for prerequisites, and there is no link...

Reply:
![prerequisites](https://user-images.githubusercontent.com/18418519/34287728-65e6e8b6-e6e9-11e7-979a-bb4f29d3d48e.png)
Leave a comment
When I display the first tutorial, the prerequisites are displayed directly to the right of step 1.